### ISSUE TITLE
Nepali Translation

### ISSUE THREAD

**Issue by Ankit8848** [rwaldron/idiomatic.js#275]
None has initiated in Nepali translation of the idiomatic JS document yet I want to contribute of this one if that's OK. :)
As I am from Nepal, I am good at speaking or writing Nepali .

**Comment by DhruvWebDev** [comment]
So you can use chatgpt to translate and contribute.

**Comment by DragonSenseiGuy** [comment]
@Ankit8848 Yes you are more than welcome to contribute. Just open a pull request!
